examples/cookbook/jotai/types.ts

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+
+import { PrimitiveAtom } from 'jotai/vanilla/atom';
+import { useHydrateAtoms } from 'jotai/utils';
+import { RenderOptions as RntlRenderOptions } from '@testing-library/react-native';
+
+type WithInitialValue<Value> = {
+  init: Value;
+};
+type UseHydrateAtomsParams = Parameters<typeof useHydrateAtoms>;
+export type InitialValues = UseHydrateAtomsParams[0];
```

Comment:
The naming of there two types is too similar `IntitialValues` `InitialValue`, could we make it more distinct. Pls add some code comment to clarify the types purposes.

Reply:
I've removed this, as it wasn't needed afterall 👍🏻